pkg/piperutils/cyclonedxBom.go

```diff
@@ -46,3 +48,22 @@ func GetBom(absoluteBomPath string) (Bom, error) {
 	}
 	return bom, nil
 }
+
+func GetPurl(filePath, bomFilename string) string {
```

Comment:
This function definition is misleading as, we don't send "filePath" to BOM. We send filePath to package manager like pom.xml or package.json.

Can we simplify in a way that, we only send file path to BOM and get back purl ?

Or rename function definition to 

GetPurl(filePathToPackageManager, realtiveBomFilePath)

Reply:
[201a9fa](https://github.com/SAP/jenkins-library/pull/5166/commits/201a9fa0be294c21389a38ef29de30ac43b2ce2a)